### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by enocol**
-Added a contact form section
- Added a user name field and an email field.
- Added a background image as displayed in Figma
- Included a form action URL to a backend called FormSpree
- Ensure all form fields and required before submission

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,5 +232,49 @@ <h1 class="sectionheading">About Myself</h1>
...
       </div>
     </section>
+    <section class="contactform-container">
+      <form
+        action="https://formspree.io/f/mwkdkrga"
+        method="post"
+        class="contactform"
+      >
+        <div class="contactheading">
+          <h1 class="c-h">Contact me</h1>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] As stated in the section requirement Kindly make sure that you follow the best practices  https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md So I recommend you to use` h1 `tag for the headline section and use `hx` tag (`h2`, `h3`, `h4`) tag by following the hierarchy  of the your HTML Code.

File: index.html [hunk 2]
```diff
@@ -232,5 +232,49 @@ <h1 class="sectionheading">About Myself</h1>
...
+        </div>
+
+        <input
+          class="nameinput"
+          type="text"
+          name="name"
+          id="username"
+          placeholder="Lorem Ipsum"
+          required
+        />
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] As stated in the project requirement Kindly make sure that **Name field accepts up to 30 chars**, Kindly Fix it.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @enocol   , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @enocol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**